neo/SmartContract/InteropService.NEO.cs

```diff
@@ -13,45 +14,47 @@
 
 namespace Neo.SmartContract
 {
-    public class NeoService : StandardService
+    static partial class InteropService
     {
-        public NeoService(TriggerType trigger, Snapshot snapshot)
-            : base(trigger, snapshot)
+        public static readonly uint Neo_Native_Deploy = Register("Neo.Native.Deploy", Native_Deploy, 0);
+        public static readonly uint Neo_Crypto_CheckSig = Register("Neo.Crypto.CheckSig", Crypto_CheckSig, 100);
+        public static readonly uint Neo_Crypto_CheckMultiSig = Register("Neo.Crypto.CheckMultiSig", Crypto_CheckMultiSig);
```

Comment:
Should it have the cost?

`Neo.Contract.Migrate` and `Neo.Contract.Create` as 0 as well?

Reply:
Maybe for `checkmultisig` is it the sum of all CheckSig?
And for migrate and create the other actions are already defined, right?

Reply:
https://github.com/neo-project/neo/blob/8412851c84336fee451d1d149c257a70484916d4/neo/SmartContract/ApplicationEngine.cs#L89-L101